README.md

```diff
@@ -46,14 +46,15 @@ c = Coverage(
         axes={
             "x": {"dataType": "float", "values": [1.23]},
             "y": {"values": [4.56]},
-            "t": {"dataType": "datetime", "values": [datetime.datetime.now()]}
+            "t": {"dataType": "datetime", "values": [datetime.now(tz=timezone.utc)]},
```

Comment:
I missed this before:
This is not the intended use of datatype, it should be left out in this case. 
See the 4th and 5th bullet in section [9.6.1.1](https://opengeospatial.github.io/ogcna-auto-review/21-069.html#_7770268a-bf17-481a-ac40-b309e60548a3). In this case `dataType` should be set to `primitive` and thus left out.

The same issue is in the example.

There is currently no enum to enforce this, as that will block custom extensions...

Reply:
Fixed